### DISCUSSION TITLE
command pimcore:generate:bundle not working

### DISCUSSION THREAD

**Discussion by mcg-pimcore** [pimcore/pimcore#10257] [Q&A / Support] [answered]
Hello,

I am facing issue to generate bundle in pimcore 10.0.6 or 10.0.4.
I cross checked env is in dev mode.
PFA for error printscreen.
![Screenshot from 2021-09-07 11-54-58](https://user-images.githubusercontent.com/87263262/132304455-5d56caba-06a0-42b7-925a-22feb2b0352e.png)

**Comment by dvesh3** [comment, marked as answer]
@mcg-pimcore please make sure that you have https://github.com/pimcore/bundle-generator bundle installed. This functionality has been moved from Core to Bundle on Pimcore X.

**Reply by mcg-pimcore** [reply]
Hello Divesh,

Thanks for reply. I will try your answer.

**Reply by mcg-pimcore** [reply]
Yes, This is a right answer. After installing bundle-generator it started working.

Followed commands:-
composer require pimcore/bundle-generator
bin/console pimcore:bundle:enable PimcoreBundleGeneratorBundle
